tests/cli/commands/test_celery_command.py

```diff
@@ -29,9 +29,6 @@
 from airflow.configuration import conf
 from tests.test_utils.config import conf_vars
 
-mock.patch('airflow.utils.cli.action_logging', lambda x: x).start()
```

Comment:
This caused a side effect. I don't know why it affected this change. Probably some session is now saving properly.